Cargo.toml

```diff
@@ -299,6 +299,14 @@ wasmtime = "18.0"
 which = "6.0.0"
 sys-locale = "0.3.1"
 
+[workspace.dependencies.windows]
```

Comment:
Is there a reason this isn't listed inline like the other dependencies?

Reply:
@kazatsuyu, don't see a need for it to be in workspace Cargo.toml just yet, should just move it gpui's Cargo.toml.

Reply:
Since the `windows` crate has a huge number of features, and the features that should be enabled for each API that we want to use are divided into smaller groups, I thought it would be more readable to separate them, since we will eventually have to specify a large number of features.

Reply:
> Since the `windows` crate has [a huge number of features](https://docs.rs/crate/windows-sys/0.52.0/features), and the features that should be enabled for each API that we want to use are divided into smaller groups, I thought it would be more readable to separate them, since we will eventually have to specify a large number of features.

I think that sounds reasonable. Thanks for clarifying!

Reply:
> @kazatsuyu, don't see a need for it to be in workspace Cargo.toml just yet, should just move it gpui's Cargo.toml.

I moved it